packages/mwp-app-route-plugin/src/index.js

```diff
@@ -20,14 +20,13 @@ export function register(
 	const ldClient = LaunchDarkly.init(options.ldkey || LAUNCH_DARKLY_SDK_KEY, {
 		offline: process.env.NODE_ENV === 'test',
 	});
-	server.expose('getFlags', memberObj => {
-		const key = (memberObj && memberObj.id) || 0;
-		return ldClient.allFlags({ ...memberObj, key, anonymous: key === 0 }).then(
+	server.expose('getFlags', (user: LaunchDarklyUser) => {
```

Comment:
I did a full text search for `getFlags` in the entire `meetup-web-components` and it looks like the only two usages are in [server-render.jsx](https://github.com/meetup/meetup-web-platform/pull/583/files#diff-d7329f1db146f21557dc646dcd0d2350). So hopefully there shouldn't be any unexpected breakages because of signature change.